src/lib/common/serial.c

```diff
@@ -54,6 +54,7 @@ void startSerial() {
   configASSERT(rval == pdTRUE);
 }
 
+// ReSharper disable once CppDFAEndlessLoop
```

Comment:
Is this some editor/code formatting thing?

Reply:
Yea, I'll delete